utils.go

```diff
@@ -195,6 +196,10 @@ func goTest(gitURL string) (pass bool, err error) {
 }
 
 func getGoDir(gitURL string) (goDir string) {
+	if len(gitURL) >= 1 && gitURL[0] == '.' {
```

Comment:
I saw this pattern earlier, we should make a helper func out of it.

Reply:
In the case of the helper func, it might be good to use the logic I posted in the earlier comment :)